src/ansi-c/expr2c.cpp

```diff
@@ -1028,6 +1027,57 @@ std::string expr2ct::convert_binary(
   const std::string &symbol,
   unsigned precedence,
   bool full_parentheses)
+{
+  if(src.operands().size()!=2)
+    return convert_norep(src, precedence);
```

Comment:
Shouldn't that now be a `PRECONDITON`?

Reply:
@kroening Would you mind fixing/commenting on this?

Reply:
expr2c has, historically, not enforced any data invariants on expressions; malformed expressions are instead printed as "Lisp" expressions.

Using a PRECONDITION here would be a very substantial change in semantics.